examples/plot_minimal_pydeseq2_pipeline.py

```diff
@@ -0,0 +1,151 @@
+"""
+PyDESeq 2 pipeline
+===========================
+
+This notebook gives a minimalistic example of how to perform DEA using PyDESeq2.
+
+It allows you to run the PyDESeq2 pipeline either on synthetic data or on one of
+the following TCGA datasets:
+
+- TCGA-BRCA
+- TCGA-COAD
+- TCGA-LUAD
+- TCGA-LUSC
+- TCGA-PAAD
+- TCGA-PRAD
+- TCGA-READ
+- TCGA-SKCM.
+
+While the synthetic data is part of this repository, TCGA data should be
+downloaded as per the instructions in `datasets/`.
+
+Running this pipeline may take a few minutes (~5-10 min for TCGA) depending on
+your setup and on the chosen dataset.
+"""
+
+import os
+import pickle as pkl
+
+from pydeseq2.DeseqDataSet import DeseqDataSet
+from pydeseq2.DeseqStats import DeseqStats
+from pydeseq2.utils import load_example_data
+
+SAVE = False  # whether to save the outputs of this notebook
+
+# %%
+# Data loading
+# ------------
+#
+# See the `datasets` readme for the required data organization.
+#
+
+DATASET = "synthetic"  # or 'TCGA-BRCA', 'TCGA-COAD', etc.
+
+OUTPUT_PATH = f"../output_files/{DATASET}"
+os.makedirs(OUTPUT_PATH, exist_ok=True)  # Create path if it doesn't exist
+
+counts_df = load_example_data(
+    modality="raw_counts",
+    dataset=DATASET,
+    debug=False,
+)
+
+clinical_df = load_example_data(
+    modality="clinical",
+    dataset=DATASET,
+    debug=False,
+)
```

Comment:
(See earlier comments about TCGA and saving)

```suggestion
counts_df = load_example_data(
    modality="raw_counts",
    dataset="synthetic",
    debug=False,
)

clinical_df = load_example_data(
    modality="clinical",
    dataset="synthetic",
    debug=False,
)
```